src/views/PreLaunchAirdrop/components/TravellerFlow/useTravellerFlow.ts

```diff
@@ -1,7 +1,11 @@
-import { useState } from "react";
+import React, { useState } from "react";
 import { Rotate, Bolt, Heart, Present } from "./TravellerFlow.styles";
 import { useHistory } from "react-router-dom";
-const TRAVELLER_FLOW_DATA = [
+const TRAVELLER_FLOW_DATA: {
+  title: string;
+  Icon: React.FC;
+  description: string | (string | { val: string; href?: string })[];
```

Comment:
So adding to my comment above, we could change this to something like
```suggestion
  description: string | React.ReactElement
```